### PR TITLE
citool tweaks

### PR DESCRIPTION
**Pull request by rabbah**
Report ongoing build cursor.
Detect job exceeded limit and report it on build completion.
Strip html tags more cleanly.

### CODE REVIEW

File: tools/build/citool [hunk 1]
```diff
@@ -100,14 +100,14 @@ def request(method, urlString, body = "", headers = {}, auth = None, verbose = F
         print 'Got response with code %s' % res.status
     return res
 
-def shell(cmd, data=None, verbose=False):
+def shell(cmd, data = None, verbose = False):
```
Comment (psuter) [inline tools/build/citool]: does your Eclipse do Python too now? if so I'm interested.

File: tools/build/citool [hunk 2]
```diff
@@ -209,14 +209,18 @@ def grepForFailingTests(args, body):
...
 def reportBuildStatus(args, body):
-    cmd = 'tail -1'
-    (time, output, error) = shell(cmd, body, args.verbose)
-    output = output.replace('</span>', '').replace('</a>', '')
-    if 'Finished: ' in output or output.startswith('Done.'):
-        print output.strip()
+    lines = body.rstrip('\n').rsplit('\n', 1)
+    if len(lines) == 2:
+        output = lines[1]
+        output = re.sub('<[^<]+?>', '', output).strip()
```
Comment (psuter) [inline tools/build/citool]: http://stackoverflow.com/questions/1732348/regex-match-open-tags-except-xhtml-self-contained-tags/1732454#1732454 (some light reading, not an actual suggestion)

**Comment by psuter** [comment]
Merged as f3511e4772029b0bf0d31b7a04356a702e65d77d

File: tools/build/citool (hunk 1 above)
Comment (rabbah) [inline tools/build/citool]: Yes - http://www.pydev.org/.

**Pull request closed without merging**